lib/compilers/wine.js

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2012-2017, Matt Godbolt
```

Comment:
Where did this come from?

Reply:
wine-cl I guess but now I stole credits ;-)